textdescriptives/about.py

```diff
@@ -0,0 +1,3 @@
+__title__ = "textdescriptives"
+__version__ = "0.1.0"  # the ONLY source of version ID
```

Comment:
seems like the pypi version is 0.1.1. This should be higher than this. so 1.0.0 or 0.2.0. I don't know exactly which one. Using 1.0.0 does say that it is stable though.

Reply:
updated to 0.2.0 in 2e63cbd